src/microbe_stage/editor/CellEditorComponent.GUI.cs

```diff
@@ -172,7 +172,7 @@ private void UpdateTotalDigestionSpeed(float speed)
     private void UpdateTotalDigestionEfficiency(float efficiency)
     {
         digestionEfficiencyLabel.Format = TranslationServer.Translate("PERCENTAGE_VALUE");
-        digestionEfficiencyLabel.Value = Mathf.RoundToInt(efficiency * 100);
+        digestionEfficiencyLabel.Value = efficiency * 100;
```

Comment:
I think we should round at most to 2 or 3 decimal places and not display like up to 8, that'd be way too much, so `Math.Round(efficiency * 100, 2)` should be used here I think. That's for two decimal places, also can do 1 or 3 decimal places.

Reply:
I rounded it to 2 decimal places since digestion speed is also rounded likewise.